packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -213,15 +213,6 @@
                         </nui-menu-action>
                     </nui-menu-group>
                 </nui-menu>
-                <button
-                    nui-button
-                    legendMenu
-                    icon="remove"
-                    type="button"
-                    *ngIf="displayDeleteButton()"
```

Comment:
why this one is deleted but not used the ngIf condition? 
shoudn't the condition be removed as well?

Reply:
I removed the button because it is not supposed to be there:
![image](https://github.com/solarwinds/nova/assets/97229167/5a668339-7e08-44ba-80fb-ac9ff2c8625a)

The displayDeleteButton() is kept because it is used on line 77 as well (if thats what you were referring to).